autoload/fern/internal/drawer/hover_popup.vim

```diff
@@ -74,7 +74,7 @@ function! s:show() abort
     call setbufline('%', 1, line)
     call helper.fern.renderer.syntax()
     call helper.fern.renderer.highlight()
-    syntax clear FernRoot
```

Comment:
I'm pretty sure this is a bug. It was giving me some trouble so I fixed it here :-)